core/frontend/src/main.ts

```diff
@@ -0,0 +1,14 @@
+import Vue from 'vue';
+import App from './App.vue';
+import router from './router';
+import store from './store';
+import vuetify from './plugins/vuetify';
+
+Vue.config.productionTip = false;
```

Comment:
why is this? I don't see any harm in having a tip in the console when running dev mode.

Reply:
Agree.